spec/services/manage_action_spec.rb

```diff
@@ -34,14 +36,69 @@
       subject
     end
 
-    it 'creates an action user' do
-      expect(subject.action_user.email).to eq('bob@example.com')
+    describe 'new action_user' do
+      it 'creates an action user' do
+        expect{ subject }.to change{ ActionUser.count }.by 1
+      end
+      
+      it 'saves available fields to action_user' do
+        action = ManageAction.create(data.merge(first_name))
+        expect(action.action_user.first_name).to eq first_name[:first_name]
+        expect(action.action_user.email).to eq data[:email]
+        expect(action.action_user.reload.first_name).to eq first_name[:first_name]
+        expect(action.action_user.reload.email).to eq data[:email]
```

Comment:
ah yeah you're right that's not how that works. it's just that I wanted to check both that the value is changed in the object returned and in the object persisted. I suppose I'll just call it to reload once and test against that.